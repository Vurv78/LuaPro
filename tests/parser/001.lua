local function test(a, b, c)
end