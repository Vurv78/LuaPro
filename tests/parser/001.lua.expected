local lupa = require("src.lib.mod")
local Node, Token = lupa.Node, lupa.Token

return Node.new(Node.Variant.Chunk, {
	Node.new(Node.Variant.Function,{
		true,
		{ { false, Token.new(Token.Variant.Identifier, "test") } },
		{
				Token.new(Token.Variant.Identifier, "a"),
				Token.new(Token.Variant.Identifier, "b"),
				Token.new(Token.Variant.Identifier, "c")
			},
		Node.new(Node.Variant.Chunk, {})
	})
})